packages/services/usage/package.json

```diff
@@ -20,10 +22,12 @@
     "dotenv": "16.4.7",
     "got": "14.4.5",
     "graphql": "16.9.0",
+    "ioredis": "5.4.2",
     "kafkajs": "2.2.4",
     "lru-cache": "11.0.2",
     "p-limit": "6.2.0",
     "pino-pretty": "11.3.0",
+    "reflect-metadata": "^0.2.0",
```

Comment:
the rest are pinned to a specific version, do we want to do the same for this package?

Reply:
I copied the line from api/package.json which was the only one that did not use a pinned version :D

I took care of it in https://github.com/graphql-hive/console/pull/6538/commits/ef77f6222544207df12d4a43a0dd1bc0431e1a22